include/posix/arpa/inet.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2019 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_POSIX_ARPA_INET_H_
+#define ZEPHYR_INCLUDE_POSIX_ARPA_INET_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <net/socket.h>
+
+static inline char *inet_ntop(sa_family_t family, const void *src, char *dst,
```

Comment:
shouldn't sa_family_t and any other structures needed here be also defined in this namespace rather than in zephyr specific headers under include/net/?

Reply:
All of zsock_* functions come from net/socket.h together with sa_family_t, to me this looks just fine.